src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
If arrow function has 1 param no need to use brackets.
Filter function is reapiting twice

```suggestion
  const men = people.filter(person => (century
    ? person.sex === 'm' && Math.ceil(person.died / 100) === century
    : person.sex === 'm'))
```